base/src/main/java/org/aya/resolve/visitor/StmtShallowResolver.java

```diff
@@ -76,7 +76,8 @@ public void resolveStmt(@NotNull Stmt stmt, @NotNull ModuleContext context) {
           var symbol = ctx.modules().get(mod).symbols().getMaybe(use.id().component(), use.id().name());
           assert symbol.isOk();   // checked in openModule
           var defVar = symbol.get();
-          var renamedOpDecl = new ResolveInfo.RenamedOpDecl(new OpDecl.OpInfo(use.asName(), use.asAssoc()));
+          var asName = use.asName().getOrDefault(use.id().name());      // TODO: probably incorrect
+          var renamedOpDecl = new ResolveInfo.RenamedOpDecl(new OpDecl.OpInfo(asName, use.asAssoc()));
```

Comment:
I am not sure whether this is correct. I have no idea about `ResolveInfo` and `RenamedOpDecl`.